_layouts/default.html

```diff
@@ -40,11 +40,14 @@ <h2 class="project-tagline">{{ page.description | default: site.description | de
       {% include anchor_headings.html html=content anchorBody="§" %}
 
       <footer class="site-footer">
-	<h3 class="project-name">k8gb is a CNCF Sandbox Project
-	</br>
-	<a href="https://www.cncf.io" target="blank">
-	    <img src="https://raw.githubusercontent.com/cncf/artwork/master/other/cncf/horizontal/black/cncf-black.svg" height="120" alt="Cloud Native Computing Foundation">
-	</a></h3>
+        <h3 class="project-name">k8gb is a CNCF Sandbox Project
+          </br>
+          <a href="https://www.cncf.io" target="blank">
```

Comment:
```suggestion
          <a href="https://www.cncf.io" target="_blank">
```

Reply:
right, although both should work, "_blank" is technically correct